examples/results2csv.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
```

Comment:
`python` doesn't exist on some systems. I tend to use `python2.7` if I know I really want Python 2. It gets more annoying for Python 3 of course, as there are multiple current versions of Python3 (but we don't support them anyway).

Ideally we would package this using distutils/setuptools and the install target would rewrite the shebang according to the python interpreter used... As discussed, not yet.